src/lib/components/ActionTable.svelte

```diff
@@ -20,6 +21,19 @@
 	let hasOutput: boolean = $derived(actions.some((a) => a.output.length > 0));
 
 	const state = getContext<LocalStore<State>>('state')?.value;
+
+	const rowClickHandler: MouseEventHandler<HTMLTableRowElement> = (e) => {
+		// Adding triple-& will increase the specificity such that it overrides
+		// the hover styling. This way when you click the row you'll immediately
+		// see the permanent highlight and don't have to mouse away.
+		const highlightClass = '[&&&]:preset-tonal-secondary';
+		const classes = e.currentTarget.classList;
+		if (!classes.contains(highlightClass)) {
```

Comment:
For future reference
https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle